### PR TITLE
人类如果想要超越光速，比鱼类生活在陆地更为滑稽。

### PR DESCRIPTION
**Pull request by lyra-planet**

### CODE REVIEW

File: packages/website/unocss.config.ts [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { presetScalpel } from 'unocss-preset-scalpel'
 import { presetScrollbar } from 'unocss-preset-scrollbar'
 import presetDaisy from 'unocss-preset-daisy'
 import { presetHeroPatterns } from '@julr/unocss-preset-heropatterns'
-
```
Comment (xhebox) [inline packages/website/unocss.config.ts:7]: Keep the empty line

File: packages/website/package.json [hunk 2]
```diff
@@ -12,19 +12,24 @@
 		"dev:remix": "remix dev"
 	},
 	"dependencies": {
-		"@babylonjs/core": "^5.23.0",
-		"@babylonjs/gui": "^5.23.0",
+		"@babylonjs/core": "^5.27.0",
+		"@babylonjs/gui": "^5.27.0",
 		"@prisma/client": "4.4.0",
+		"@remix-run/node": "^1.7.2",
```
Comment (xhebox) [inline packages/website/package.json:18]: Replace `@remix-run/node` with `@remix-run/server-runtime`

File: packages/website/app/routes/home.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Link, Outlet } from "@remix-run/react";
+import type { LinksFunction,MetaFunction } from '@remix-run/node'
```
Comment (xhebox) [inline packages/website/app/routes/home.tsx:2]: You have removed `@remix-run/node` from `package.json`, but not here.

Comment (lyra-planet) [reply packages/website/app/routes/home.tsx]: ojbk

**Review by xhebox — APPROVED** [review]

**Pull request merged**